tests/dialects/test_riscv.py

```diff
@@ -89,6 +90,43 @@ def test_comment_op():
     assert code == "    # my comment\n"
 
 
+def test_label_op_without_comment():
+    label_str = "mylabel"
+    label_op = riscv.LabelOp(label_str)
+
+    assert label_op.label.data == f"{label_str}"
+
+    code = riscv_code(ModuleOp([label_op]))
+    assert code == f"{label_str}:\n"
+
+
+def test_label_op_with_comment():
+    label_str = "mylabel"
+    label_op = riscv.LabelOp(f"{label_str}", comment="my label")
+
+    assert label_op.label.data == "mylabel"
+    assert label_op.label.data == f"{label_str}"
```

Comment:
```suggestion
    assert label_op.label.data == label_str
```
Am I missing something? similarly in the rest of the file